lib/pages/new_private_chat/widget/expansion_contact_list_tile.dart

```diff
@@ -26,103 +28,132 @@ class ExpansionContactListTile extends StatelessWidget {
+                ),
+                const SizedBox(
+                  width: 8.0,
                 ),
-              ),
-              const SizedBox(
-                width: 12.0,
-              ),
-              Expanded(
-                child: Column(
-                  mainAxisSize: MainAxisSize.min,
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  children: [
-                    IntrinsicWidth(
-                      child: Row(
-                        children: [
-                          Expanded(
-                            child: Row(
-                              children: [
-                                Flexible(
-                                  child: BuildDisplayName(
-                                    profileDisplayName:
-                                        snapshot.data?.displayName,
-                                    contactDisplayName: contact.displayName,
-                                    highlightKeyword: highlightKeyword,
-                                    style: Theme.of(context)
-                                        .textTheme
-                                        .titleMedium
-                                        ?.copyWith(
-                                          color: Theme.of(context)
-                                              .colorScheme
-                                              .onSurface,
-                                        ),
-                                  ),
+                Expanded(
+                  child: Column(
+                    mainAxisSize: MainAxisSize.min,
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      Padding(
+                        padding: const EdgeInsets.only(bottom: 4),
+                        child: IntrinsicWidth(
+                          child: Row(
+                            children: [
+                              Expanded(
+                                child: Row(
+                                  children: [
+                                    Flexible(
+                                      child: BuildDisplayName(
+                                        profileDisplayName:
+                                            snapshot.data?.displayName,
+                                        contactDisplayName: contact.displayName,
+                                        highlightKeyword: highlightKeyword,
+                                        style: LinagoraTextStyle.material()
+                                            .bodyMedium2
+                                            .copyWith(
+                                              color: Theme.of(context)
+                                                  .colorScheme
+                                                  .onSurface,
+                                              fontWeight: FontWeight.w600,
+                                            ),
```

Comment:
why not move it to System design ?

Reply:
please move it to the System design and use it in: chat list item, contact list item,